util/configure

```diff
@@ -346,6 +349,12 @@ for my $opt (@ARGV) {
         }
         $path = File::Spec->rel2abs($path);
         push @ngx_opts, "--with-$lib=$path";
+        $with_ext_lib{$lib} = 1;
+
+    } elsif ($opt =~ /^--with-(openssl|pcre|zlib)-opt=(.*)/) {
+        my ($lib, $opt) = ($1, $2);
+        push @ngx_opts, "--with-$lib-opt=-g $opt";
+        $with_ext_lib_opts{"$lib-opt"} = 1;
```

Comment:
We don't need to add the `-opt` suffix to the hash table key. When we look up this table, we use the key of `$lib` anyway. It's wasteful to always append this artificial `-opt` suffix for no gains.